.ci/infrastructure-docker-build.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 PROFILE=$1
+BASE_IMAGE=$2
```

Comment:
This line and its associated `if` block is never being called/used in the current repo as the second argument is never passed. Is this being used in any other repos? If not, why is this required?